ordering/utils.py

```diff
@@ -115,6 +116,42 @@ def _handle_screening_day_error(episode_list):
     if one_is_arrow and one_is_flash and both_are_episode_17:
         _swap_episode_rows(episode_list, 78, 79)
 
+def _handle_air_time_error(episode_list):
+    # handles when two episodes air on the same day
+    seasons = ['', 'Midseason', 'Midseason', 'Midseason', 'Midseason', 'Midseason', 'Midseason',
+               'Summer', 'Summer', 'Summer', 'Fall', 'Fall', 'Fall']
+    air_orders = {'Fall 2016' : [(LEGENDS_OF_TOMORROW, VIXEN)],
+                  'Midseason 2017' : [(FLASH, LEGENDS_OF_TOMORROW)],
+                  'Fall 2017' : [(FLASH, LEGENDS_OF_TOMORROW)],
+                  'Midseason 2018' : [(FLASH, BLACK_LIGHTNING)],
+                  'Fall 2018' : [(ARROW, LEGENDS_OF_TOMORROW),
+                                 (FLASH, BLACK_LIGHTNING),
+                                 (SUPERGIRL, BLACK_LIGHTNING)],
+                  'Midseason 2019' : [(ARROW, BLACK_LIGHTNING),
+                                      (LEGENDS_OF_TOMORROW, ARROW)],
+                  'Fall 2019' : [(BATWOMAN, SUPERGIRL),
+                                 (FLASH, ARROW)]}
+
+    i=0
+    while i < len(episode_list)-1:
```

Comment:
Instead of using a while loop here it'd be better to use a for loop. They're simpler to read/understand and aren't susceptible to infinite looping issues.

This should achieve the same result:
```suggestion
    for i in range(len(episode_list)):
```

Reply:
yeah true, I’ll make the changes later today since this will slightly affect the logic in the loop